lib/generation/generators/value_objects/generation_configuration/pb_generation_configuration.dart

```diff
@@ -112,10 +112,10 @@ abstract class GenerationConfiguration {
         await _setMainScreen(
             tree.rootNode, '${tree.name.snakeCase}/${fileName}.dart');
       }
-      _commitImports(tree.rootNode, tree.name.snakeCase, fileName);
-
       await _iterateNode(tree.rootNode);
 
+      await _commitImports(tree.rootNode, tree.name.snakeCase, fileName);
```

Comment:
Switching the order of _iterateNode() and _commitImports() has no side effects on generation? Just checking 🤔

Reply:
Not that I am aware of, the only reason we had it the other way was so we could `replace` the import, but since now we are injecting the actual import to the PBGenCache, then it can generate all the necessary imports at once

Reply:
Sounds good, thanks!